modules/Routing.js

```diff
@@ -28,6 +29,10 @@ function createRoute(element, parentRoute, namedRoutes) {
     options.willTransitionTo = createTransitionToHook(props.to, props.params, props.query);
     props.path = props.path || props.from || '*';
   } else {
+    invariant(
+      React.Children.count(props.children) || props.handler !== RouteHandler,
+      "A route without children must specify a handler that isn't RouteHandler"
```

Comment:
The idea behind my wording was for it to make sense to someone who omitted handlers entirely, and therefore got `RouteHandler` as default. Your phrasing is better, but might be confusing in this case?